ar/transaction.py

```diff
@@ -170,8 +170,9 @@ def tobytes(self):
     @property
     def api_url(self):
         return self.peer.api_url
+    
     @api_url.setter
-    def set_api_url(self, api_url):
+    def api_url(self, api_url):
```

Comment:
pawan paudel is saying that the set_ prefix here is wrong, and throws an error. There is a working function at https://github.com/xloem/pyarweave/blob/d7d41ec35c05cdc403cfbd021820d169bd04068a/ar/bundle.py#L122 that uses the same naming convention as pawan paudel's, so this is likely a needed change and should be merged.